test/powershell/Language/Classes/scripting.Classes.NestedModules.tests.ps1

```diff
@@ -54,72 +53,74 @@ Describe 'NestedModules' -Tags "CI" {
             'class A { [string] foo() { return "A"} }',
             'class B { [string] foo() { return "B"} }'
         ) -Content 'class C { [string] foo() { return "C"} }'
+    }
+
+    AfterAll {
+        $env:PSModulePath = $originalPSModulePath
+        Get-Module @('ABC', 'NoRoot', 'WithRoot') | Remove-Module
```

Comment:
```suggestion
        Remove-Module 'ABC', 'NoRoot', 'WithRoot'
```

Reply:
That emits an error if the module isn't found, which doesn't happen for the former. Either we need `-ErrorAction SilentlyContinue` or we should keep the original version.

Reply:
I agree to keep original.